src/Aws/Laravel/AwsServiceProvider.php

```diff
@@ -28,7 +24,14 @@
  */
 class AwsServiceProvider extends ServiceProvider
 {
-    const VERSION = '1.1.0';
+    const VERSION = '1.5.0';
```

Comment:
That's a good point. @phazei maybe this should be maintained as a separate fork rather than merged in.